.ocamlformat

```diff
@@ -1,4 +1,4 @@
-version=0.20.1
```

Comment:
Do you know if we can edit the configuration to void the extra space in the indentation “

Reply:
Nevermind, I like the new version better :)